src/main/java/frc/robot/RobotContainer.java

```diff
@@ -102,76 +96,76 @@ private void initSubsystems() {
-            //CDSForwardCommand = new CDSForwardCommand(CDSSubsystem, shooterSubsystem);
-            //CDSReverseCommand = new CDSReverseCommand(CDSSubsystem, shooterSubsystem);
-            //CDSSubsystem.getAllianceColor();
-            break;
-          }
+          case "DriveBaseSubsystem":
+            {
+              System.out.println("Drivebase enabled");
+              driveBaseSubsystem = new DriveBaseSubsystem(driverJoystick, false);
+              break;
+            }
+          case "CDSSubsystem":
+            {
+              System.out.println("CDS enabled");
+              CDSSubsystem = new CDSSubsystem();
+              break;
+            }
           case "IntakeSubsystem":
-          {
-            System.out.println("Intake enabled");
-            intakeSubsystem = new IntakeSubsystem(); 
-            break;
-          }
+            {
+              System.out.println("Intake enabled");
+              intakeSubsystem = new IntakeSubsystem();
+              break;
+            }
           case "ShooterSubsystem":
-          {
-            System.out.println("Shooter enabled");
-            shooterSubsystem = new ShooterSubsystem();
-            break;
-          }
+            {
+              System.out.println("Shooter enabled");
+              shooterSubsystem = new ShooterSubsystem();
+              break;
+            }
           case "LimelightSubsystem":
-          {
-            System.out.println("Limelight enabled");
-            limelightSubsystem = new LimelightSubsystem();
-            break;
-          }
+            {
+              System.out.println("Limelight enabled");
+              limelightSubsystem = new LimelightSubsystem();
+              break;
+            }
           case "ClimbSubsystem":
-          {
-            System.out.println("Climb enabled");
-            
-            climbSubsystem = new ClimbSubsystem();
-            climbUPCommand = new ClimbUPCommand(climbSubsystem);
-            climbDOWNCommand = new ClimbDOWNCommand(climbSubsystem);
-            break;
-          }
+            {
+              if (!Constants.oneController) {
+                climbSubsystem = new ClimbSubsystem(operatorJoystick);
+                climbCommand = new ClimbCommand(climbSubsystem);
+                System.out.println("Climb enabled");
+              }
+              break;
+            }
         }
       }
     }
   }
 
   private void initCommands() {
-    //Initializes commands based on enabled subsystems
-    if(driveBaseSubsystem != null){
+    // Initializes commands based on enabled subsystems
+    if (driveBaseSubsystem != null) {
       driveBaseTeleopCommand = new DriveBaseTeleopCommand(driveBaseSubsystem);
       driveBaseSubsystem.setDefaultCommand(driveBaseTeleopCommand);
     }
     if(CDSSubsystem != null && shooterSubsystem != null){
-      CDSForwardCommand = new CDSForwardCommand(CDSSubsystem, shooterSubsystem);
+      CDSForwardCommand = new CDSForwardCommand(CDSSubsystem);
```

Comment:
Your new command doesn't get instantiated. And shooterSubsystem is no longer a requirement. Try something like this to make the auto advance the default CDS command:

```   
if(CDSSubsystem != null){
      CDSForwardCommand = new CDSForwardCommand(CDSSubsystem);
      CDSSubsystem.setDefaultCommand(new CDSAutoAdvanceCommand(CDSSubSystem));
}
```